.github/workflows/release.yaml

```diff
@@ -160,14 +160,14 @@ jobs:
       - name: Wait for images to be available
         run: |
           echo "Waiting for images to be pullable from registry..."
-          
+
           images='${{ needs.get-managed-components.outputs.managed_images }}'
           read -a IMAGE_ARRAY <<< "$images"
-          
+
           MAX_RETRIES=120
           SLEEP_INTERVAL=60
           retry_count=0
-          
+
           for image in "${IMAGE_ARRAY[@]}"; do
             image_url="${DOCKER_REGISTRY}/${image}:${COMPONENT_TAG}"
```

Comment:
Here we also need to wait for stig-fips tag if the component is stig fips. We've encountered during the last run, that the job would wait for `<sriov-daemon-fips>:<non-fips-tag>`